dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/LeadLag.mo

```diff
@@ -0,0 +1,55 @@
+within Dynawo.NonElectrical.Blocks.Continuous;
+
+/*
+* Copyright (c) 2015-2021, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+model LeadLag "Simple lead-lag filter"
+
+  import Modelica;
+  import Dynawo;
+  import Dynawo.Types;
+
+  extends Modelica.Blocks.Interfaces.SISO(y(start = y_start));
+
+  parameter Types.PerUnit K = 1 "Gain";
+  parameter Types.Time T1 "Lead time constant";
```

Comment:
t1, t2 instead of T1, T2 ? @adriguir what do you think ?

Reply:
t1, t2